WriteHeader/calibration_tool.py

```diff
@@ -0,0 +1,105 @@
+from PIL import Image as image
+from PIL.TiffTags import TAGS
+import numpy
+import os
+import SimpleITK as sitk
+
+class CalibrationTool:
+    """Read and store calibration data from camera calibration"""
+
+    def __init__(self):
+        self.file = []
+
+        self.principle_dist = 0
+        self.x_offset = 0
+        self.y_offset = 0
+        self.pixel_size = 0
+
+        self.xray_position = []
+        self.image_normal = []
+        self.image_up = []
+
+        self.camera_path = 0
+        self.image_all = []
+
+        self.direction_matrix = []
+
+    def read_file(self, base_dir, file_dir, filename):
+        """Given filename, read all lines in file"""
+
+        path = base_dir + file_dir + filename
+        open_file = open(path, "r")
+        self.file = open_file.readlines()
+
+    def parse_file(self):
+        """Parse and clean all values from the calibration file"""
+
+        self.principle_dist = self.file[1].strip()
+        self.x_offset = float(self.file[2].strip())
+        self.y_offset = float(self.file[3].strip())
+        self.pixel_size = float(self.file[4].strip())
+
+        self.xray_position = self.file[6].strip().split()
+        self.image_normal = self.file[7].strip().split()
+        self.image_up = self.file[8].strip().split()
+
```

Comment:
I suggest to convert them to float right here rather than later.

Reply:
It will help for other developers such as me to use directions and variables right away by calling the corresponding functions.